docs/callbacks-promises-events.rst

```diff
@@ -12,11 +12,11 @@ we provide multiple ways to act on asynchronous functions.
 Most web3.js objects allow a callback as the last parameter, as well as returning promises to chain functions.
 
 Ethereum as a blockchain has different levels of finality and therefore needs to return multiple "stages" of an action.
-To cope with requirement we return a "promiEvent" for functions like ``web3.eth.sendTransaction`` or contract methods.
-This "promiEvent" is a promise combined with an event emitter to allow acting on different stages of action on the blockchain, like a transaction.
+To cope with requirement we return a "PromiEvent" for functions like ``web3.eth.sendTransaction`` or contract methods.
+This "PromiEvent" is a promise combined with an event emitter to allow acting on different stages of action on the blockchain, like a transaction.
```

Comment:
Maybe rephrase L15-L16 and L18-L19 as:

These stages are encapsulated into a "PromiEvent", which combines a promise with an event emitter. The event emitter fires an event for each of the finality stages.

An example of a function that benefits from PromiEvents is `web3.eth.sendTransaction`. The event emitter inside the promise fires events such as ``receipt`` and ``transactionHash``.